### DISCUSSION TITLE
Is it possible to define a replicable part?

### DISCUSSION THREAD

**Discussion by SamueleFurnari** [OrchardCMS/OrchardCore#9904] [Q&A] [answered]
Hello everybody!

I wish to define a content type with a field for wich I can add multiple instances during content item definition.

I'll try to explain the requirement with an example.

I have defined a content type Process, with a Text field and a complex field (that I defined as Part).

The Part is composed by:
- a Text Field
- a Content Picker Field that allow user to select an item of type Process.

I wish to allow user to define a process that had a title and an unpredictable number of instances of complex content type, a sort of _named part_ with plus sign to add more instances.

Is this possibile?

Thanks

**Comment by ns8482e** [comment, marked as answer]
Create Process Type with `BagPart` and Create another content type `ProcessItem`  that could use complex part .
Select `ProcessItem` as contained content type of the `BagPart`.

**Reply by SamueleFurnari** [reply]
Thank you very much!
